### ISSUE TITLE
export @forward, deprecate Lazy.@forward

### ISSUE THREAD

**Issue by tpapp** [MikeInnes/MacroTools.jl#74]
Sometimes I just depend on `Lazy` for `Lazy.@forward`, but the latter is already in `MacroTools`, although unexported.

Perhaps it would make sense to move it to this package "officially", deprecating `Lazy.@forward`. The functionality is not strongly tied to either package, but this would remove one dependency from those who just need this single macro.

**Comment by aleferna12** [comment]
I agree, @forward is very useful and it would be nice to skip the Lazy dependency

**Comment by cstjean** [comment]
I'm down for that, but OTOH users who are `using` both Lazy and MacroTools will get a bunch of breakage. So on the whole, I'm not sure that it's worth it... `using MacroTools: @forward` seems acceptable...
